<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateCommitmentQWidget</class>
 <widget class="QWidget" name="CreateCommitmentQWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create a Commitment</string>
  </property>
  <widget class="QFrame" name="createCommitmentQFrame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>451</width>
     <height>451</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="commitmentNameQLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>180</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Commitment Name:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="commitmentNameQLineEdit">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>40</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="dateStartQLabel">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>110</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Start Date</string>
    </property>
   </widget>
   <widget class="QLabel" name="dateEndQLabel">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>110</y>
      <width>70</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>End Date</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateStartQDateEdit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>130</y>
      <width>110</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEndQDateEdit">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>130</y>
      <width>110</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="qtyQLabel">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>170</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Qty</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="qtyQLineEdit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>190</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="unitsLeftQLabel">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>170</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Units</string>
    </property>
   </widget>
   <widget class="QLabel" name="perQLabel">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>190</y>
      <width>21</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>per</string>
    </property>
   </widget>
   <widget class="QLabel" name="unitsRightQLabel">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>170</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Units</string>
    </property>
   </widget>
   <widget class="QComboBox" name="dropDownUnitsRightQComboBox">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>190</y>
      <width>79</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>Day</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Week</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Month</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="taskQLabel">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>240</y>
      <width>31</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Task:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="dropDownTaskQComboBox">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>240</y>
      <width>91</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>Writing</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Compose Music</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="listenersQLabel">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>280</y>
      <width>58</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Listeners:</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkKeyboardQCheckBox">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>300</y>
      <width>83</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Keyboard</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkMicQCheckBox">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>320</y>
      <width>83</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Mic</string>
    </property>
   </widget>
   <widget class="QComboBox" name="dropDownAudioListenerQComboBox">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>320</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>Built-In Mic</string>
     </property>
    </item>
   </widget>
   <widget class="QCheckBox" name="makeDefaultQCheckBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>360</y>
      <width>231</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Make this my default commitment</string>
    </property>
   </widget>
   <widget class="QPushButton" name="createCommitmentQPushButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>390</y>
      <width>150</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Create Commitment</string>
    </property>
   </widget>
   <widget class="QPushButton" name="backQPushButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>390</y>
      <width>150</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <widget class="QComboBox" name="dropDownUnitsLeftQComboBox">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>190</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>Minute(s)</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Hour(s)</string>
     </property>
    </item>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
